runtime/doc/eval.txt

```diff
@@ -6894,7 +6895,7 @@ match({expr}, {pat} [, {start} [, {count}]])			*match()*
 			GetText()->match('word')
 			GetList()->match('word')
 <
-				*matchadd()* *E798* *E799* *E801* *E957*
+				*matchadd()* *E290* *E798* *E799* *E801* *E957*
```

Comment:
interestingly nvim uses E5031 here instead

seems that originally this error didn't have an error code, so it was given one in this commit: https://github.com/neovim/neovim/commit/fb07391ce46b8bff90ef7ef073b75919a307e64c that also added more info to the emsg. Vim noticed later on and assigned it to E290 :eyes: